### ISSUE TITLE
Security issues for XML and Image Handlers

### ISSUE THREAD

**Issue by cristianadam** [copperspice/copperspice#73]
Have a look at http://lists.qt-project.org/pipermail/development/2018-September/033573.html 
It seems Qt4 has had some security issues being reported, which might affect CopperSpice.

**Comment by agserm** [comment]
Thanks so much for the heads up! If you have any other information about this issue, please let us know, and of course pull requests are welcome. We will investigate this and see how to best address the security issues identified here.

**Comment by agserm** [comment]
I took a look at the referenced thread on the mailing list, and I have been unable to find patches or work related to the issues mentioned. The only vulnerability I can find is https://nvd.nist.gov/vuln/detail/CVE-2016-10040, which is still unresolved in Qt 4 and Qt 5.  In fact, I found https://stackoverflow.com/questions/18898807/reading-and-validating-xml-document-with-external-entities-using-qt which suggests that the QXmlSimpleReader is deprecated. If you have any further details about unpatched issues, we are very interested in keeping CopperSpice current with all security updates. Thanks for your help!
